orchestrator/modules/tools/discovery/action_semantic_index.py

```
@@ -119,19 +119,80 @@ async def ensure_indexed(
+        narrowing is an optimization; it must never cost more than it saves
+        (observed 37–67s/call when the OpenRouter embedding upstream degrades).
+        """
+        try:
+            cached = self._cache.get_embeddings_batch([query], model=model_key).get(query)
+        except Exception:
+            cached = None
+        if cached:
+            return cached, True, False
+
+        task = asyncio.ensure_future(self._embedding_manager.generate_embedding(query))
+        done, pending = await asyncio.wait({task}, timeout=timeout_s)
```

Ruff (0.15.20)

[warning] Unpacked variable `done` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)